package.json

```
@@ -3,7 +3,7 @@
   "version": "5.4.0",
   "description": "ESLint plugin about ECMAScript syntactic features.",
   "engines": {
-    "node": ">=8.10.0"
+    "node": ">=14"
```

```suggestion
    "node": "^14.18.0 || >=16.0.0"
```

Can we make it this instead? See eg. this thread: https://twitter.com/voxpelli/status/1521885437001932807

Then we can know that ESM is supported (`await import()`) and that we can use `node:` prefixes when importing built in modules

---

I think we should keep this in sync with ESLint core tbh

```suggestion
    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

---

Thanks for the review!

Since this package is never used by ESLint itself, I agree that it doesn't have to be the same engine that ESLint supports.

However, I just noticed that it may be used by eslint-plugin-n in the future.
https://github.com/eslint-community/eslint-plugin-n/blob/2ce285db073df4669e64d4ebf4e7179dd4d2a13e/package.json#L17

I need to change the supported node version to `">=12.22.0"`.
https://github.com/eslint-community/eslint-plugin-n/blob/2ce285db073df4669e64d4ebf4e7179dd4d2a13e/package.json#L6

---

By the way, eslint-plugin-n is planning to drop support for Node<v14.
https://github.com/eslint-community/eslint-plugin-n/issues/42